libr/include/r_userconf.h.acr

```diff
@@ -39,7 +39,7 @@
 #define R2_LIBDIR "@LIBDIR@"
 #define R2_INCDIR "@INCLUDEDIR@/libr"
 #define R2_DATDIR "@DATADIR@"
-#define R2_WWWROOT R2_DATDIR "/radare2/" R2_VERSION "/www"
+#define R2_WWWROOT "share/radare2/" R2_VERSION "/www"
```

Comment:
why kill datdir? share can be overriden from configure

Reply:
Because DATADIR is absolute (`/usr/share`) and can't be used with R2_PREFIX